doc/Command-Reference.md

```diff
@@ -738,6 +765,59 @@ This command displays the history of the previous reboots up to 10 entry
   2020_10_09_04_53_58  warm-reboot  Fri Oct  9 04:51:47 UTC 2020  admin
   ```
 
+**show reboot-cause history all**
+
+This command displays the history of the previous reboots up to 10 entry of the Switch and the DPUs for which the midplane interfaces are up.
+
+- Usage:
+  ```
+  show reboot-cause history all
+  ```
+
+- Example:
+  ```
+  root@MtFuji:~# show reboot-cause history all
+  Device    Name                 Cause                                      Time                             User    Comment
+  --------  -------------------  -----------------------------------------  -------------------------------  ------  -------
+  SWITCH    2024_07_23_23_06_57  Kernel Panic                               Tue Jul 23 11:02:27 PM UTC 2024  N/A     N/A
```

Comment:
@rameshraghupathy can we rename "NPU" instead of "SWITCH" ?

Reply:
@prgeor ok, renaming "NPU" instead of "SWITCH